### PR TITLE
CHDEXCHAE-64 fix Docker Compose in tests

### PR DESCRIPTION
**Pull request by ThrawnCA**

### CODE REVIEW

**Comment by ThrawnCA** [comment]
Comment searching is fixed.

File: ckanext/data_qld/actions.py [hunk 1]
```diff
@@ -447,6 +447,8 @@ def list_datarequests(original_action, context, data_dict):
         return result
 
     query = data_dict.get("q", "")
+    if not query:
+        return result
```
Comment (duttonw) [inline ckanext/data_qld/actions.py:451]: No sort order?

File: ckanext/data_qld/actions.py [hunk 2]
```diff
@@ -470,24 +472,15 @@ def list_datarequests(original_action, context, data_dict):
...
-
-        thread_data = get_comment_thread(entity_id, content_type)
-        comments_data = get_comments_data_for_index(thread_data)
-
-        if query.lower() in comments_data.lower():
-            datarequest_ids.append(entity_id)
-
-    return datarequest_ids
+    search_pattern = '%{}%'.format(query)
+    threads = model.Session.query(CommentThread.url) \
```
Comment (duttonw) [inline ckanext/data_qld/actions.py:476]: Nice

**Review by duttonw — APPROVED** [review]
Unsure about exit early. Better to return early inside the text search instead.

File: ckanext/data_qld/actions.py (hunk 1 above)
Comment (duttonw) [reply ckanext/data_qld/actions.py:451]: re-reviewed, this is 'adding' to the original calls requests so no need to sort a already sorted list.

Comment (ThrawnCA) [reply ckanext/data_qld/actions.py:451]: The original function can handle that.

**Pull request merged**